torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -354,6 +354,11 @@ void RRefContext::delPendingUser(const ForkId& forkId) {
   TORCH_INTERNAL_ASSERT(
       iter != pendingUsers_.end(),
       "Inconsistent states: attempt to delete a non-exist UserRRef.");
+  users_.emplace(
```

Comment:
Why inserting it here instead of in `createUserRRef`?

Reply:
@mrshenli 

I think we should store confirmed user rrefs. Do you have a reason for tracking un-confirmed user rrefs?

Reply:
Makes sense, as only confirmed RRefs need to send delete message (it might be different when we add fault tolerant later). In that case, shall we call the map `confirmedUsers_` to be explicit?